app/src/main/java/org/fossasia/phimpme/sharetwitter/LoginActivity.java

```diff
@@ -41,23 +40,25 @@
 import twitter4j.conf.Configuration;
 import twitter4j.conf.ConfigurationBuilder;
 
-public class LoginActivity extends Activity {
+public class LoginActivity extends ThemedActivity {
 
 	public static final int TWITTER_LOGIN_RESULT_CODE_SUCCESS = 1111;
 	public static final int TWITTER_LOGIN_RESULT_CODE_FAILURE = 2222;
 
 	private static final String TAG = "LoginActivity";
 
 	private WebView twitterLoginWebView;
-	private AlertDialog mAlertBuilder;
 	private static String twitterConsumerKey;
 	private static String twitterConsumerSecret;
 
 	private static Twitter twitter;
 	private static RequestToken requestToken;
+	AlertDialog dialog;
+	AlertDialog.Builder progressDialog;
+
 
 	@Override
-	protected void onCreate(Bundle savedInstanceState) {
+	public void onCreate(Bundle savedInstanceState) {
```

Comment:
@the-dagger Yes, because I extend LoginActivity to ThemeActivity so it is showing an error.

Error:(61, 17) error: onCreate(Bundle) in LoginActivity cannot override onCreate(Bundle) in ThemedActivity
attempting to assign weaker access privileges **('protected')** ; was public